### PR TITLE
Docstring fix

### PR DESCRIPTION
**Pull request by ayman-saleh**
Correcting docstring for `add_image_with_boxes` method. Fixed spelling mistake.

### CODE REVIEW

**Review by xuhdev — APPROVED** [review]

**Comment by xuhdev** [comment]
Thanks!

File: torch/utils/tensorboard/writer.py [hunk 1]
```diff
@@ -579,7 +579,7 @@ def add_image_with_boxes(self, tag, img_tensor, box_tensor, global_step=None,
             dataformats (string): Image data format specification of the form
               NCHW, NHWC, CHW, HWC, HW, WH, etc.
         Shape:
-            img_tensor: Default is :math:`(3, H, W)`. It can be specified with ``dataformat`` agrument.
+            img_tensor: Default is :math:`(3, H, W)`. It can be specified with ``dataformats`` argument.
```
Comment (vishwakftw) [inline torch/utils/tensorboard/writer.py]: nit: with the ```:attr:`dataformats` ``` argument

Comment (xuhdev) [reply torch/utils/tensorboard/writer.py]: @vishwakftw  `:attr` shouldn't be added because it is used for denoting class attributes. See #25885

Comment (vishwakftw) [reply torch/utils/tensorboard/writer.py]: I thought that for functions, the arguments could use :attr: in the doc strings

Comment (vishwakftw) [reply torch/utils/tensorboard/writer.py]: But I see that this is a function in a class, please regard my comment.

Comment (ayman-saleh) [reply torch/utils/tensorboard/writer.py]: Im sorry, I don't understand, should I change \`\`dataformats\`\` argument to :attr:\`dataformats\` argument?

Comment (ayman-saleh) [reply torch/utils/tensorboard/writer.py]: ops, just read the last couple comments, my bad.

**Comment by ayman-saleh** [comment]
@xuhdev , just out of curiosity (This is my first pull request) what is the process to go from approval to a merge?

**Comment by xuhdev** [comment]
Just wait :)

@pytorchbot merge this please

**Pull request closed without merging**